opengapps-files.mk

```diff
@@ -2,11 +2,22 @@
 PRODUCT_COPY_FILES += $(call gapps-copy-to-system,all,etc)
 PRODUCT_COPY_FILES += $(call gapps-copy-to-system,all,framework)
 
+# check if we are building a vendor image
+ifneq ($(CALLED_FROM_SETUP),true)
+BUILD_VENDORIMAGE := $(shell CALLED_FROM_SETUP=true BUILD_SYSTEM=build/core \
+      command make --no-print-directory -f build/core/config.mk dumpvar-BOARD_VENDORIMAGE_FILE_SYSTEM_TYPE)
+endif
+
```

Comment:
Note that command to get the value of `BOARD_VENDORIMAGE_FILE_SYSTEM_TYPE`
is taken from the definition of the function `get_build_var` in `build/envsetup.sh`.